conf/testOnlyDoNotUseInAppConf.routes

```diff
@@ -12,11 +12,11 @@
 # Add all the application routes to the prod.routes file
 # microservice specific routes
 
-GET        /check-income-tax/test-only/update-income/delete/:id                                                testOnly.controllers.TaiUpdateIncomeController.delete(id: Int)
+GET        /test-only/check-income-tax/test-only/update-income/delete/:id                                                testOnly.controllers.TaiUpdateIncomeController.delete(id: Int)
```

Comment:
tai-fe calls the routes you changed in tai-be - so you need to change here too: https://github.com/hmrc/tai-frontend/blob/57b4ccceef14b556ac0ff71061708816e3470eb1/app/testOnly/connectors/TaiConnector.scala#L36 

I'm not sure whether you need to change the fe routes - are there warnings in tai-fe? If you do need to change them then could you remove the second test-only from the routes?